updates/1.139/190923_pedagogy_form_history_type.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
+
+  <sql>alter table pedagogyformhistory add column type varchar(255) not null;</sql>
+  <sql>update pedagogyformhistory set type = "EDIT" where type = null;</sql>
```

Comment:
This won't update anything. Because type is a not null column, values for previous rows will be empty strings. Can simply be...

`update PedagogyFormHistory set type='EDIT';`

Also capitalize the table name in the previous statement (might be that Linux/MariaDB is stricter about this).